dissect/ntfs/c_ntfs.py

```diff
@@ -564,6 +564,25 @@
 ACE_OBJECT_FLAGS = c_ntfs.ACE_OBJECT_FLAGS
 COLLATION = c_ntfs.COLLATION
 
+IO_REPARSE_TAG_CLOUD = (
```

Comment:
I believe you will need to manually add all of these constants to the `.pyi` file for type checkers to pick up their existence.

Reply:
Waiting on this.